sxcu/sxcu.py

```diff
@@ -122,9 +141,13 @@ def upload_file(
         if self_destruct:
             data["self_destruct"] = ""
         url = join_url(self._get_api_endpoint(default_domain=False), "/files/create")
-        with open(file, "rb") as img_file:
-            files = {"file": img_file}
+        if isinstance(file, io.BytesIO):
+            files = {"file": file}
             res = request_handler.post(url, files=files, data=data)
+        else:
+            with open(file, "rb") as img_file:
+                files = {"file": img_file}
+                res = request_handler.post(url, files=files, data=data)
```

Comment:
I don't like this approach of calling `requests.post` in two places. Instead you could make them use a single call to `requests.post`

Reply:
yeah i don't like it either but im not exactly sure how to get around it. If you'd like to you should be able to make the changes yourself and commit to my branch.

Reply:
I have done: https://github.com/naveen521kk/sxcu/pull/90/commits/550a33a40002f6b494ada55b297009be67594f68